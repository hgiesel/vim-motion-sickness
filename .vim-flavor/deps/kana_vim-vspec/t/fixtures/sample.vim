describe 'Syntax highlighting'
  before
    set ignorecase
  end
  after
    set ignorecase&
  end
  it 'highlights vspec-specific keywords'
    Expect type(s) ==# type('')
    Expect type(s) not ==# type(0)
    Expect expr { {'': '}'} } to_throw
    SKIP
    TODO
  end
  context 'with a nested suite'
    it 'is supported'
    end
  end
end
