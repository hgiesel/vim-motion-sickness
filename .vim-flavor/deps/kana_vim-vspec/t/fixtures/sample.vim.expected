cccccccc SSSSSSSSSSSSSSSSSSSSS
  cccccc
    ... ..........
  ccc
  ccccc
    ... ...........
  ccc
  cc SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
    cccccc ......._OOO_.....SS.
    cccccc ......._ooo_OOO_.....N.
    cccccc llll_...SS..SSS..._________
    cccc
    cccc
  ccc
  ccccccc SSSSSSSSSSSSSSSSSSSSS
    cc SSSSSSSSSSSSSS
    ccc
  ccc
ccc
